planner_reasoner/lib/pddl_generator.pl

```diff
@@ -0,0 +1,479 @@
+:- module(pddl_generator, [
+    generate_pddl/2
+]).
+
+:- use_module(library(semweb/rdf11)).
+:- use_module(library(semweb/rdfs)).
+
+generate_pddl(Resource, Stream) :-
```

Comment:
FEATURE: generate PDDL file from either domain or problem `Resource` into the `Stream`